geographic_msgs/GeoPoint center  # The center of the cylinder, altitude is disregarded.
float64 height  # Height of the cylinder in m from sea level.
float64 radius  # Radius of the cylinder in m.
